kt/entry-generation/godot-kotlin-symbol-processor/src/main/kotlin/godot/annotation/processor/ext/ksFunctionDeclarationExt.kt

```diff
@@ -8,14 +8,19 @@ import godot.entrygenerator.model.ConstructorAnnotation
 import godot.entrygenerator.model.FunctionAnnotation
 import godot.entrygenerator.model.RegisteredConstructor
 import godot.entrygenerator.model.RegisteredFunction
+import java.io.File
 
-internal fun KSFunctionDeclaration.mapToRegisteredConstructor(): RegisteredConstructor {
+internal fun KSFunctionDeclaration.mapToRegisteredConstructor(
+    isFqNameRegistrationEnabled: Boolean,
```

Comment:
You are passing the same 3 arguments around everywhere, I would gather them in a single data class and pass a single parameter to make things less bloated.

Reply:
Agreed. Will change that.